Library/Homebrew/service.rb

```diff
@@ -323,10 +349,34 @@ def to_plist
       base[:StandardErrorPath] = @error_log_path if @error_log_path.present?
       base[:EnvironmentVariables] = @environment_variables unless @environment_variables.empty?
 
+      if keep_alive?
+        if @keep_alive[:always].present?
+          base[:KeepAlive] = @keep_alive[:always]
+        elsif @keep_alive.key? :succesful_exit
+          base[:KeepAlive] = { SuccessfulExit: @keep_alive[:succesful_exit] }
+        elsif @keep_alive.key? :crashed
+          base[:KeepAlive] = { Crashed: @keep_alive[:crashed] }
+        elsif @keep_alive.key? :path
+          base[:KeepAlive] = { PathState: @keep_alive[:path].to_s }
+        end
```

Comment:
```suggestion
        if (always = @keep_alive[:always].presence)
          base[:KeepAlive] = always
        elsif @keep_alive.key?(:successful_exit)
          base[:KeepAlive] = { SuccessfulExit: @keep_alive[:succesful_exit] }
        elsif @keep_alive.key?(:crashed)
          base[:KeepAlive] = { Crashed: @keep_alive[:crashed] }
        elsif @keep_alive.key?(:path)
          base[:KeepAlive] = { PathState: @keep_alive[:path].to_s }
        end
```

which of these keys are you still going to set if they are either `false` or `nil` (separate questions)?

Reply:
All except path for `false`, for `nil` I doubt there is a use.

Reply:
Any remaining comments based on this?

Reply:
Nah, find as-is.